### PR TITLE
mgmt, bump tsp version in emitter-package.json

### PR DESCRIPTION
**Pull request by XiaofeiCao**
# Description

Mainly tsp-arm, for fixing `ManagedServiceIdentity`.

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#developer-guide)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: eng/emitter-package-lock.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "TempTypeSpecFiles",
-  "lockfileVersion": 3,
+  "lockfileVersion": 2,
```
Comment (weidongxu-microsoft) [inline eng/emitter-package-lock.json:3]: I am not sure why but there is lots of diff in lock file. Is it npm version causing it?

Comment (weidongxu-microsoft) [reply eng/emitter-package-lock.json:3]: It should still follow your local nodejs/npm

Not sure but I guess you need to have 18/9

Comment (XiaofeiCao) [reply eng/emitter-package-lock.json:3]: I'm using 20/8. Let me update my npm to 9.

Comment (XiaofeiCao) [reply eng/emitter-package-lock.json]: Regen by 20/10.

**Review by weidongxu-microsoft — APPROVED** [review]

**Pull request merged**